yaas/blueprint.py

```diff
@@ -1,137 +1,54 @@
 # coding: utf-8
 
+""" Interact with Ambari blueprints. """
+
 from __future__ import absolute_import
 from __future__ import print_function
 
-import argparse
-import inspect
 import json
 import pprint
 import requests
 import sys
 
-from . import common
-
-
-def command(parser, args):
-    """ Interact with Ambari blueprints. """
-    subparsers = parser.add_subparsers(dest='subcommand')
-    subcommands = {
-        'add': _add,
-        'list': _list,
-        'remove': _remove,
-        'show': _show,
-        }
-    for name, fn in subcommands.items():
-        subparsers.add_parser(name, help=inspect.getdoc(fn))
-    subargs, extra = parser.parse_known_args(args)
-    subcommands[subargs.subcommand](
-        subparsers.choices[subargs.subcommand],
-        extra)
+from . import config
+from . import utils
 
-
-def _add(parser, args):
+def add(blueprint, blueprint_name, validate_topology=True):
```

Comment:
Well, my initial thought would be they would just use something like `self.server` or `self.port`.

> The only drawback is that yaas can only have one server configured per process that imports yaas. With a yaas.Server object, you could have multiple servers floating around.

I think this is the biggie... but again, that's an additional feature, not a flaw with this PR.

> Either way, I highly doubt people are writing a multi-threaded script to interact with multiple ambari servers that they're managing.

should/could :p